### PR TITLE
feat: add H3 block

### PR DESCRIPTION
**Pull request by holazz**
- Refactor `H1Block.vue` and `H2Block.vue` to `HeadingBlock.vue`
- add H3 option
- add placeholder style

### CODE REVIEW

**Comment by greentfrapp** [comment]
LGTM thanks for the refactoring!  🧴

File: src/components/blocks/HeadingBlock.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  if (props.block.type === BlockType.H1) return 'Heading 1'
+  else if (props.block.type === BlockType.H2) return 'Heading 2'
+  else if (props.block.type === BlockType.H3) return 'Heading 3'
+})
+
+const sizeClass = computed(() => {
+  if (props.block.type === BlockType.H1) return 'text-4xl'
+  else if (props.block.type === BlockType.H2) return 'text-3xl'
+  else if (props.block.type === BlockType.H3) return 'text-2xl'
+})
```
Comment (VitorEstevam) [inline src/components/blocks/HeadingBlock.vue:36]: Don`t you think we can use a map to return an object with these values? Kinda like a style configuration

Comment (VitorEstevam) [inline src/components/blocks/HeadingBlock.vue:36]: dont u think having a map that returns an object with these values can be a better approach to implement the other headings without adding many lines and unnecessary complexity? 

something like this:
``` javascript
var configs= {
    BlockType.H1: {
        size: "text-4xl",
        placeholder: "Heading 1"
    },
    BlockType.H2: {
        size: "text-3xl",
        placeholder: "Heading 2"
    },
};
```

Comment (greentfrapp) [reply src/components/blocks/HeadingBlock.vue:36]: Just updated to something similar actually!

**Pull request merged**